doc/installation/README.md

```diff
@@ -16,18 +16,20 @@ also be found in the links above.
 
 | Supported Platform                         |Latest Stable                 |
 |--------------------------------------------|------------------------------|
-|  Windows (x64)                             |[.whl][whl-win-x64]           |
-|  Windows (x86)                             |[.whl][whl-win-x86]           |
-|  macOS 10.12+                              |[.whl][whl-macos]             |
+|  Windows (x64)                             |[.whl][whl]           |
+|  Windows (x86)                             |[.whl][whl]           |
+|  macOS 10.12+                              |[.whl][whl]             |
+|  Linux (Python Wheel)                      |[.whl][whl]             |
 |  Ubuntu 14.04+                             |[.deb][deb]                   |
 |  Debian 8.7+                               |[.deb][deb]                   |
 |  CentOS 7+                                 |[.rpm][rpm]                   |
 |  Red Hat Enterprise Linux 7+               |[.rpm][rpm]                   |
 |  openSUSE                                  |[.rpm][rpm]                   |
 |  Fedora 25+                                |[.rpm][rpm]                   |
 
-[deb]: https://packages.microsoft.com/ubuntu/14.04/prod/pool/main/m/mssql-cli/mssql-cli_0.15.0-1_all.deb
-[rpm]: https://packages.microsoft.com/rhel/7/prod/mssql-cli-0.15.0-1.el7.x86_64.rpm
-[whl-win-x64]: https://files.pythonhosted.org/packages/79/31/1be42f3632a30bb126e02cda4312b797f5f2cdef60b9c62596196a475037/mssql_cli-0.15.0-py2.py3-none-win_amd64.whl
-[whl-win-x86]: https://files.pythonhosted.org/packages/21/8c/9829c2094b4f179f9c2f4548b2ba089867f84a3758cb70575ef2a905e877/mssql_cli-0.15.0-py2.py3-none-win32.whl
-[whl-macos]: https://files.pythonhosted.org/packages/43/5d/c9af6aec5b491e7b0c5ccf00b4b8062282d6c4cfb4c0417891bd6013e299/mssql_cli-0.15.0-py2.py3-none-macosx_10_11_intel.whl
+
+[deb]: https://packages.microsoft.com/ubuntu/16.04/prod/pool/main/m/mssql-cli/mssql-cli_1.0.0-1_all.deb
```

Comment:
One issue is these files don't exist yet until we publish, so when we merge the vbump into master, the links won't work. I'm leaning towards just hand editing the links after we push to pypi and the other repos.